src/components/GridLayout.js

```diff
@@ -0,0 +1,118 @@
+//import '../App.css';
+import dynamic from 'next/dynamic';
+import Image from 'next/image';
+import {useState} from 'react';
+
+//import {Suspense} from 'react';
+
+import useThemeStore from '../hooks/useThemeStore';
+//import useThemeStorePersist from '../hooks/useThemeStorePersist';
+//TODO als svg einbinden
+import brightnessDay from '../images/brightness-day.png';
+import brightnessNight from '../images/brightness-night.png';
+import closeBox from '../images/close-box.png';
+import crossHairsOff from '../images/MapButtons_2020_off.png';
+//import crossHairsOn from '../images/MapButtons_2020_on.png';
+
+//import Map from './Map';
+
+const Map = dynamic(() => import('./Map'), {
+	ssr: false,
+	suspense: true,
+});
+
+export default function GridLayout() {
+	//const [zoom, setZoom] = useState(8);
```

Comment:
Please remove unused code in comments

Zoom with GoogleMaps should work without controlling it with a state yourself. There should be no need for a logic, that calls `setZoom`